portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,14 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function addRandomIntro() {
+  const intros =
+      ['Student', 'Developer', 'Woman', 'Human'];
+
+  // Pick a random greeting.
+  const intro = intros[Math.floor(Math.random() * intros.length)];
+
+  // Add it to the page.
+  const introContainer = document.getElementById('intro-container');
+  introContainer.innerText = intro;
```

Comment:
These two variables are dynamic. You are overwriting the values each time addRandomIntro is called with something different.

Please use var instead.

https://google.github.io/styleguide/javascriptguide.xml#var

Reply:
I changed it for addRandomGreeting too